spec/currency_spec.rb

```diff
@@ -77,6 +77,18 @@
     end
   end
 
+  describe "#hash" do
+    specify "equal currencies from different loaders have the same hash" do
+      currency_from_first_loader = currency
+
+      Money::Currency.class_variable_set(:@@loaded_currencies, {})
+      currency_from_second_loader = Money::Currency.new('USD')
```

Comment:
After chatting on Slack, I believe the following test setup would achieve the same without having to reach into the gem internals to reset the loader?
```ruby
currency_1 = Money::Currency.find('USD')
currency_2 = YAML.load(Money::Currency.find('USD').to_yaml)
```

Reply:
Nice!  I'll give that a try!

Reply:
Done.